<launch>

    <group ns="planner">

<!--        <remap from="openni_points" to="/zed/point_cloud/cloud_registered"/>-->
        <remap from="map" to="/rtabmap/proj_map"/>
        <remap from="move_base_simple/goal" to="/planner_goal"/>

        <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
            <rosparam file="$(find path_planning)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
            <rosparam file="$(find path_planning)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
            <rosparam file="$(find path_planning)/params/costmap_local_params.yaml" command="load" />
            <rosparam file="$(find path_planning)/params/costmap_global_params.yaml" command="load" />
<!--            <rosparam file="$(find path_planning)/params/base_local_planner_params.yaml" command="load" />-->
        </node>
    </group>

</launch>